doc/content/microreactors/hpmr_h2/simba_results.md

```diff
@@ -0,0 +1,66 @@
+| 8	        | 1.21%	    | -0.37%	    | -1.38%	    | -0.11% |
+| 9	        | 1.36%	    | -0.24%	    | -1.37%	    | -0.01% |
+| 10	        | 1.47%	    | -0.22%	    | -1.18%	    | 0.09% |
+| 11	        | 1.56%	    | -0.23%	    | -1.18%	    | 0.12% |
+| 12	        | 1.43%	    | -0.11%	    | -1.15%	    | 0.12% |
+| 13	        | 1.56%	    | -0.11%	    | -1.18%	    | 0.16% |
+| 14	        | 1.38%	    | -0.19%	    | -1.26%	    | 0.04% |
+| 15	        | 1.23%	    | -0.12%	    | -1.19%	    | 0.04% |
+| 16	        | 1.22%	    | -0.01%	    | -1.11%	    | 0.11% |
+| 18	        | 3.10%	    | 2.79%	    | 1.55%	    | 2.54% |
+| 19	        | 0	        | 0	        | 0	        | 0	    |
+| 20	        | 0	        | 0	        | 0	        | 0	    |
+| RMSE	    | 1.66%	    | 1.41%	    | 1.75%	    | 1.31% |
+
+In that table, Assembly 1 is any of the center fuel assembly and Assemblies 2 and 3 correspond to any peripheral fuel assembly surrounded by 4 and 3 fuel assemblies, respectively. The relative errors in power are shown for each axial 10-cm thick axial layer of each assembly type and for all of them combined (Total), with axial layers 1 and 20 corresponding to the bottom and top of the core, respectively. The values are 0 for the top and bottom two layers because they do not contain any fissile material. The last row reports the Root Mean Square (RMSE) of the error in the corresponding assembly types.
+
+
+## Coupled Solution
+
+[simba_results1] (a) and (b) show the 3-D temperature and the $YH_x$ stoichiometric ratio spatial distributions in the moderator pins, respectively. These figures reveal the stoichiometric ratio and moderator temperature to be inversely proportional. In fact, the hydrogen stoichiometric ratio increases when the moderator temperature decreases and vice-versa. This is further illustrated in [simba_results1] (c), which reports the radially averaged moderator temperature and hydrogen concentration as a function of the distance from the bottom of the moderator pins.
+
+
+!media hpmr_h2/simba_results1.jpeg
+    caption= Three-dimensional temperature spatial distribution in the moderator pins, (b) 3-D hydrogen stoichiometric ratio spatial distribution, and (c) radially averaged moderator temperature and hydrogen stoichiometric ratio for 16 10-cm-high axial levels as a function of the distance from the bottom of the YHx moderator pins.
+    id=simba_results1
+    style=width:80%; margin-left:auto; margin-right:auto
+
+[table-floating2] reports the average, minimum, and maximum value for fuel temperature, denoted as $T_f$, moderator temperature, $T_m$, and, stoiciometric ration, $c_H$.
+
+!table id=table-floating2 caption=Maximum, minimum, and average values for the fuel temperature, moderator temperature, and hydrogen stoichiometric ratio.
+| Variable | Average | Minimum | Maximum |
+| ---------| ------- | ------- | ------- |
+| $T_f$, K | 954.02  | 912.74  | 978.52  |
+| $T_m$, K | 947.83  | 911.11  | 973.84  |
```

Comment:
crazy how close fuel and moderator are.. are we overestimating how good Heat Pipes are ?

or is Tf somewhat homogenized?

Reply:
The fuel and moderator pins are very close to each other and the temperature gradient is mainly in the axial direction - which is why the fuel and moderator temperatures are so close to each other. Do you have any concern about these results?

Reply:
@GiudGiud , my guess is that this closeness is caused by the lack of gaps around the fuel and heat pipes, (i.e.,fuel and heat pipes are in perfect contact with the monolith). Gaps are explicitly modeled only around the moderator pins. since we were mainly interested in the hydrogen redistribution and leakage in the gap.

Reply:
the temperature is just very homogeneous in the reactor overall (70 k difference).  So maybe how close these values are should be considered in that light